docs/api.md

```diff
@@ -51,6 +51,7 @@
 .. autofunction:: scores.probability.murphy_thetas
 .. autofunction:: scores.probability.roc_curve_data
 .. autofunction:: scores.probability.brier_score
+.. autofunction:: scores.probability.ensemble_brier_score
```

Comment:
I like this name, but crps has crps_for_ensemble and so that pattern may be more consistent for readers

Reply:
I think this is a fair comment. If I were scrolling down an alphabetically ordered list of scoring names, I'd probably prefer to see brier at the front here. Having said that, there are lots of ways of searching, and I'm anticipating the brier_score docstring references this version for ensembles